lib/commands/gestures.js

```diff
@@ -515,6 +477,43 @@ export async function macosPressAndDragAndHold (
   });
 };
 
+/**
+ * Raise invalid argument error if element id was unset or x and y were unset.
+ * @param {string} [elementId]
+ * @param {number} [x]
+ * @param {number} [y]
+ * @returns {void}
+ */
+function requireElementIdOrXY (elementId, x, y) {
+  if (!_.isString(elementId) && !_.isNumber(x) || !_.isNumber(y)) {
```

Comment:
the `or` condition must be in parentheses

Reply:
or it must be similar to the below one:  `!_.isString(elementId) && !(_.isNumber(x) && _.isNumber(y))`